provider/junit5/src/main/kotlin/au/com/dius/pact/provider/junit5/PactJUnit5VerificationProvider.kt

```diff
@@ -155,12 +155,15 @@ class PactVerificationExtension(
 
   override fun getDisplayName(invocationIndex: Int): String {
     return when {
-      pactSource is BrokerUrlSource && pactSource.result != null -> if (pactSource.result!!.pending) {
-        pactSource.result!!.name + " [PENDING]"
-      } else {
-        pactSource.result!!.name
+      pactSource is BrokerUrlSource && pactSource.result != null -> {
+        var displayName = pactSource.result!!.name + " - ${interaction.description}"
+        if (pactSource.tag.isNotEmpty()) displayName += " (tag ${pactSource.tag})\""
+        if (pactSource.result!!.pending) {
+          "$displayName [PENDING]"
+        } else {
+          displayName
+        }
       }
-      pactSource is BrokerUrlSource && pactSource.tag.isNotEmpty() -> "${pact.consumer.name} - ${interaction.description} (tag ${pactSource.tag})"
```

Comment:
We still need this case, because the pactSource.result value is only populated from the new broker endpoint. For people with an older broker, it will be null.

Reply:
Added the case back!